django/thunderstore/api/cyberstorm/serializers/common.py

```diff
@@ -0,0 +1,6 @@
+from rest_framework import serializers
+
+
+class CyberstormDynamicLinksSerializer(serializers.Serializer):
```

Comment:
This actually shouldn't exist right now, because it's not used anywhere. I'll remove it, but keep ur comment in mind when we add it this at some point 👍